src/packages/@ncigdc/uikit/SparkMeter.js

```diff
@@ -10,10 +10,12 @@ export default ({ value, width = 30, max = 30 }) =>
       display: 'inline-block',
       pointerEvents: 'none',
     }}
+    {...props}
```

Comment:
Should merge this styles with `props.style`
Also either destructure style or move the prop spread to the top.